eng/pipelines/templates/jobs/ci.yml

```diff
@@ -74,9 +74,16 @@ jobs:
           MatrixConfigs: ${{ parameters.MatrixConfigs }}
           MatrixFilters: ${{ parameters.MatrixFilters }}
           MatrixReplace: ${{ parameters.MatrixReplace }}
+          SparseCheckoutPaths: [ "sdk/", ".vscode"]
           CloudConfig:
             Cloud: Public
           AdditionalParameters:
             ServiceDirectory: ${{ parameters.ServiceDirectory }}
             Artifacts: ${{ parameters.Artifacts }}
             TestProxy: ${{ parameters.TestProxy }}
+          ${{ if eq(parameters.ServiceDirectory, 'auto') }}:
+            EnablePRGeneration: true
+            PreGenerationSteps:
+              - template: /eng/common/pipelines/templates/steps/save-package-properties.yml
```

Comment:
I don't believe so. For context, I just merged a PR to eng/common that enables generation of a much smarter unit test matrix. For standard builds, it won't look any different. If we were to touch every service directory in the repo, it will smartly batch up the unit tests so you still get full coverage without assigning more than 10 packages to each test job. This logic only applies to PRs, your `internal` builds will remain exactly the same as they are now.

What this is doing particular change is doing is ensuring that we get the package properties saved before generating the matrix, so we can take advantage of that new smarter logic.